src/backends/torch/torchlib.cc

```diff
@@ -722,6 +722,11 @@ namespace dd
                       .to(_main_device);
           }
       }
+    if (ad_mllib.has("reg_weight"))
+      {
+        _reg_weight = ad_mllib.get("reg_weight").get<double>();
```

Comment:
`yolox_reg_weight` ? I believe we want it in the `net` object ?

Reply:
It can go in the `net` object
Maybe this parameter can be useful for other detection models that have a balance between regression & classification? e.g fasterrcnn or retinanet

Reply:
It'd go into `net` anyways no ?